www/js/app.js

```diff
@@ -0,0 +1,45 @@
+// Ionic Starter App
+
+// angular.module is a global place for creating, registering and retrieving Angular modules
+// 'starter' is the name of this angular module example (also set in a <body> attribute in index.html)
+// the 2nd parameter is an array of 'requires'
+// 'starter.controllers' is found in controllers.js
+angular.module('starter', ['ionic', 'starter.controllers'])
```

Comment:
Is the name of your app 'starter'? That's what `starter` here means. You'll also have to rename your controllers: `homechef.controllers`, etc.